tests/integration_tests/reports/commands_tests.py

```diff
@@ -339,7 +339,7 @@ def create_report_slack_chart_with_csv():
         report_schedule = create_report_notification(
             slack_channel="slack_channel",
             chart=chart,
-            report_format=ReportDataFormat.DATA,
+            report_format=ReportDataFormat.CSV,
```

Comment:
@cemremengu Thanks for the pr! Can you also include some tests like these that also use `XLSX` as the data format?